test/unit_tests/braket/circuits/test_angled_gate.py

```diff
@@ -44,3 +46,11 @@ def test_getters():
 @pytest.mark.xfail(raises=AttributeError)
 def test_angle_setter(angled_gate):
     angled_gate.angle = 0.14
+
+
+def test_np_float_angle_json():
+    angled_gate = AngledGate(angle=np.float32(0.15), qubit_count=1, ascii_symbols=["foo"])
+    assert (
+        BaseModel.construct(target=[0], angle=angled_gate.angle).json()
+        == '{"target": [0], "angle": 0.15000000596046448}'
```

Comment:
Is this approximation guaranteed?

Reply:
To add on to this, rather than asserting the JSON string, can you convert it back and assert the round-trip value is the same? That way, we don't have to hardcode values, and we test the important piece of functionality.

Reply:
I believe so. Nevertheless, I replaced the equality check with a regex match and used the np isclose() method to compare the two values.